lib/resources/nginx_module.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+# author: Rony Xavier, rx294@gmail.com
+# author: Aaron Lippold, lippold@gmail.com
+
+module Inspec::Resources
+  class NginxModule < Inspec.resource(1)
+    name 'nginx_module'
```

Comment:
Should we call this `nginx_modules`? We are reading out multiple modules, but give one module as argument